tests/test_runner/models.py

```diff
@@ -4,7 +4,19 @@
 class Person(models.Model):
     first_name = models.CharField(max_length=20)
     last_name = models.CharField(max_length=20)
-    friends = models.ManyToManyField("self")
+    friends = models.ManyToManyField("self", through="PersonFriend")
+
+class PersonFriend(models.Model):
+    from_person = models.ForeignKey(
+        Person, on_delete=models.CASCADE, related_name="friends_from"
```

Comment:
nit: rename `friends_from` to `from_person` for consistency. The same for `friends_to` below